pkg/feature/templates/servicemesh/kserve/envoy-oauth-temp-fix.tmpl

```diff
@@ -0,0 +1,76 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: envoy-oauth-temp-fix-before
+  namespace: {{ .ControlPlane.Namespace }}
+  labels:
+    opendatahub.io/related-to: OSSM-4873
```

Comment:
This ticket is already closed. I wonder if we still would need this?

Reply:
@aslak can chime in here, but we have to support previous versions of OSSM?

Reply:
The Issue ref is why it's there, not where it will be cleaned up. I don't think we technically have a clean up task defined.

Reply:
I guess we can leave it here, just in case somebody is using older OSSM.

BTW, I didn't understand your comment @aslakknutsen :smile:

Reply:
@israel-hdez The reason we have these filters is because of OSSM-4873. OSSM support a range of versions as @bartoszmajsak said, so until the cause for this has been pushed outside the support range we need it.

My previous answer was just that the issue used in the label is the cause of this filter and not the issue/task used to track the removal of it. My confusion. :)